bounties/Code Sample For Airgapped Wallet.md

```diff
@@ -0,0 +1,115 @@
+
+**Proposed programming language:** Python
+
+## Overview
+
+<!--
+Please provide the context required to complete the bounty.
+
+Questions you should answer here:
+1. What is the high level explanation of this bounty? (1-3 sentences)
+2. What problem is this solving?
+3. What are the requirements for this solution?
+-->
+
+**Discussion:** https://github.com/XRPLBounties/Proposals/discussions/47 (Must Read for details)
+
+## Milestones
+
+<!--
+Please split the bounty into smaller milestones with individual awards in the following template.
+The first milestone should be the core functionality, while the rest can be useful add-ons.
+
+| # | High-Level Description | Details | Proposed Potential Award |
+| 1 | ... | ... | $ |
+
+(The proposed amounts from milestones should add up to the amount listed at the top of the bounty proposal)
+-->
+
+
+| # | High-Level Description | Details | Proposed Potential Award |
+| --- | --- | --- | --- |
+| 1 | Keypair/Wallet management system | Generate an XRPL account and store it securely with a password feature built-in using a cryptography algorithm | $1,500 |
+| 2 | XRP Payments | Decrypt the previously stored keypair on the filesystem by password and sign a Payment transaction | $1,500 |
+| 3 | Transmit & Relay Transaction | **Efficiently** & **Securely** transfer a signed transaction blob from the airgapped machine and relay the Payment to a rippled node and verify Payment's validity  | $1,000 |
+| 4 | Tutorial & Descriptions & Diagrams (R&D) | Research and write a proper documentation on the code sample's directory and explain all the components, its purpose and how it all works together to form a reliable & secure way to store a keypair | $2,000 |
+
+
+The value of the proposal is determined by the amount of research and development put in place, researching & developing the code sample takes a considerable amount of time since the airgapped wallet's effectiveness is not determined by the code itself (to secure the keypair), it's the design and security practices behind the concept which gatekeeps the keypair without any way of compromising it.
+
+Simplifying this to new developers on the ecosystem will give them a sense of why security is prioritized in applications which hold custody to cryptocurrencies & assets. 
+
+All milestones on this Bounty must be fully reached with synchronization  since with one single feature missing (aka. 1 milestone not being accomplished), it won't work as an airgapped wallet requires all the proposed mechanics to work together:
+1. Generate wallet and store securely (using cryptography in this case)
+2. Sign a transaction and keep the fields complete (i.e. the last ledger sequence is high enough to keep up with ledger progress [+100 advancement])
+3. Transmit the signed transaction blob in a secure channel *without* compromising or connecting the airgapped machine to a network (i.e using a QR code)
+4. Relay the transaction to a rippled node and display the results
```

Comment:
Should a requirement for the completion of this bounty be having the final tutorial be merged into xrpl.org?

Reply:
Yes. It would be uploaded as a code sample with tutorials/descriptions/diagrams under the [code sample](https://github.com/XRPLF/xrpl-dev-portal/tree/master/content/_code-samples) directory.